AIMSICD/src/main/java/com/SecUpwN/AIMSICD/utils/Status.java

```diff
@@ -27,11 +25,11 @@
 
     public enum Type {
```

Comment:
The About page contains 6 statuses, so this enum should contain the same 6 statuses, shouldn't it?